Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1022,6 +1022,16 @@ LayoutUnit RenderGrid::masonryContentSize() const
     return m_masonryLayout.gridContentSize();
 }
 
+Vector<LayoutRect> RenderGrid::masonryItems()
```

Comment:
I think we should rename this function since in its current form it's kinda misleading. We are not actually returning the actual masonry items to the caller but their `LayoutRects` so we should make that clear

Reply:
Thanks. Fixed.